201-vm-lb-winrm-windows/azuredeploy.json

```diff
@@ -0,0 +1,574 @@
+{
+    "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "adminUsername": {
+            "type": "string",
+            "metadata": {
+                "description": "Admin username"
+            }
+        },
+        "adminPassword": {
+            "type": "securestring",
+            "metadata": {
+                "description": "Admin password"
+            }
+        },
+        "dnsLabelPrefix": {
+            "type": "string",
+			"metadata": {
+				"description": "Unique DNS Name for the Public IP used to access the Virtual Machine."
+			}
+        },
+        "vmNamePrefix": {
+            "type": "string",			
+            "defaultValue": "myVM",
+			"metadata": {
+				"description": "Virtual machine name prefix for primary vm instances"
+			}
+        },
+        "nvmNamePrefix": {
+            "type": "string",			
+            "defaultValue": "mytestVM",
+			"metadata": {
+				"description": "Virtual machine name prefix for secondary vm instances"
+			}
+        },
+        "lbName": {
+            "type": "string",			
+            "defaultValue": "myLB",
+			"metadata": {
+				"description": "Load Balancer name"
+			}
+        },
+        "nicNamePrefix": {
+            "type": "string",			
+            "defaultValue": "nic",
+			"metadata": {
+				"description": "Network interface card name prefix fro primary instances"
+			}
+        },
+        "nicNamePrefixN": {
+            "type": "string",
+            "defaultValue": "nicN",
+			"metadata": {
+				"description": "Network interface card name prefix fro secondary instances"
+			}
+        },
+        "publicIPAddressName": {
+            "type": "string",			
+            "defaultValue": "myPublicIP",
+			"metadata": {
+				"description": "Public ip address name"
+			}
+        },
+        "vnetName": {
+            "type": "string",			
+            "defaultValue": "myVNET",
+			"metadata": {
+				"description": "Virtual network name"
+			}
+        },
+        "imagePublisher": {
```

Comment:
Here we are not restricting the user with a few options. User can enter of his choice in case he wants an OS other than 2012 R2. If I make it choice, again I need to update whenever a new OS is available.